cdragontoolbox/__main__.py

```diff
@@ -429,6 +442,14 @@ def create_parser():
     subparser.add_argument('bin',
                            help="BIN file to extract")
 
+    # skn files commands
+
+    subparser = subparsers.add_parser('skn-extract',
+                                      help="Extract an SKN file to a folder")
```

Comment:
For consistency with other commands:
- `extract` (lowercase)
- `folder` -> `directory`